tests/.infra/crw-ci/dashboard-next/pr-check/Jenkinsfile

```diff
@@ -217,43 +202,64 @@ pipeline {
                     fi
 
                     git checkout $ghprbSourceBranch
-                    
-
-                    echo "Build the 'che-dashboard-next' image"
+                """
 
-                    docker build . -f apache.Dockerfile -t docker.io/maxura/che-dashboard:${mutableCheImageTag}
+                withCredentials([string(credentialsId: 'ed71c034-60bc-4fb1-bfdf-9570209076b5', variable: 'maxura_docker_password')]) {
+                    sh """
+                        echo "Build 'che-dashboard' image"
+                        echo "${maxura_docker_password}" | docker login -u maxura --password-stdin
+                        
+                        cd ${WORKSPACE}/che-dashboard
+                        docker build . -f apache.Dockerfile -t quay.io/crw_pr/che-dashboard:${mutableCheImageTag}
+                    """
+                }
 
-                    docker push docker.io/maxura/che-dashboard:${mutableCheImageTag}
+                withCredentials([string(credentialsId: 'quay.io-crw_pr+crw_pr_bot_user_token', variable: 'QUAY_TOKEN')]) {
+                    sh """
+                        echo "Push 'che-dashboard' image"
+                        echo "${QUAY_TOKEN}" | docker login -u="crw_pr+crw_pr_bot" --password-stdin quay.io/crw_pr
 
-                    cd ..
-                """
+                        echo y | docker push quay.io/crw_pr/che-dashboard:${mutableCheImageTag}
```

Comment:
Why are you pushing to `quay.io/crw_pr` instead of `quay.io/eclipse`?

Reply:
@benoitf have expressed concerns about pollution of tags and risk of promoting images that may have security issues on official repo, and I have agreed with him.